source/docs/software/mecanum-drive.rst

```diff
@@ -4,22 +4,22 @@ Programming Tutorial - Mecanum Drivetrain
 Mecanum Physics
 ---------------
 
-A very common choice for drivetrain in FTC is :term:`mecanum drive <Mecanum Wheel>`, but we have noticed that not a lot of people know how they work or how to properly program them.  :term:`Mecanum wheels <Mecanum Wheel>` have rollers at a 45° angle to the rest of the wheel.
+The :term:`mecanum drive <Mecanum Wheel>` is a very popular drive train in FTC. The mecanum drive enables holonomic movement. This means that the drivetrain is able move in all directions. Forward, backward, side to side, translating while rotating, etc. `Here is a neat video <https://www.youtube.com/watch?v=pP8ajNMx84k>`_ demonstrating such movement.
```

Comment:
```suggestion
:term:`Mecanum drive <Mecanum Wheel>` is a very popular drive train in FTC. Mecanum drive enables holonomic movement. This means that the drivetrain is able move in any direction while rotating: forwards, backwards, side to side, translating while rotating, etc. `Here is a neat video <https://www.youtube.com/watch?v=pP8ajNMx84k>`_ demonstrating such movement.
```

The mecanum drive sounds like it refers to a specific drivetrain.

Holonomic drive is not simply being able to translate in any direction, it's translating in any direction with heading being independent.